lib/takeaway.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'menu'
+require_relative 'custy'
+require 'twilio-ruby'
+
+class Takeaway
+   
+   
+   def initialize
+      @custy_orders = []
+   end
+   
+   def total(custy)
+       "#{custy.name} your order comes to £#{custy.total} thanks for shopping at t-quay"
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)